### ISSUE TITLE
Add version dropdown in doc website

### ISSUE THREAD

**Issue by nkakouros** [ansible/ansible#45651]
##### SUMMARY

Through search engines, bookmarks, history i often end up in docs for different ansible versions. A dropdown for instance in the top right corner of the page would allow changing the version easily.

##### ISSUE TYPE
- Feature Idea

##### ADDITIONAL INFORMATION

Also, it would be handy to have the chosen version be saved in a cookie and have the browser automatically redirect the user to their selected version or at least display a warning or notice with a link to their chosen version.

**Comment by sivel** [comment]
cc @acozine

**Comment by acozine** [comment]
@tterranigma thanks for submitting a detailed ticket. Trouble navigating multiple versions of the docs is a known challenge. We removed the version dropdown when we did a major reorganization of the documentation (after which many pages appeared only in one version of the documentation) early this year. A better way to navigate among versions is on the docket, though it isn't currently the docs team's top priority. I'm going to keep this ticket open, so when we get to this (may not be for a few months), I'll have your info handy.

Related to #37049.
